sql/hive/src/main/scala/org/apache/spark/sql/hive/HiveSessionState.scala

```diff
@@ -32,10 +32,19 @@ import org.apache.spark.sql.streaming.StreamingQueryManager
 
 /**
  * A class that holds all session-specific state in a given [[SparkSession]] backed by Hive.
- * @param catalog A Hive client used for interacting with the metastore.
- * @param analyzer An analyzer that uses the Hive metastore.
- * @param plannerCreator Lambda to create a [[SparkPlanner]] that converts optimized logical
- *                       plans to physical plans.
+ * @param sparkContext The [[SparkContext]].
+ * @param sharedState The shared state.
+ * @param conf SQL-specific key-value configurations.
+ * @param experimentalMethods The experimental methods.
+ * @param functionRegistry Internal catalog for managing functions registered by the user.
+ * @param catalog Internal catalog for managing table and database states.
```

Comment:
Add comment on difference from `SessionCatalog`: `HiveSessionCatalog` uses Hive client for interacting with metastore.